src/main.js

```diff
@@ -18,7 +18,7 @@ if (!errorOnNoMatch && customErrorMessage) {
 const environmentYaml = library.getCurrentEnvironmentVars();
 const environmentDictionary = library.buildEnvironmentDictionary(input_scope, inputYaml, environmentYaml, errorOnNoMatch);
 
-console.log('Scoped Variables:', environmentDictionary);
+console.log('Scoped Variables:', environmentDictionary); // This doesn't print right with JSON.stringify()
```

Comment:
Is JSON.stringify() what one would normally call in this situation? Is the assumption that someone might looks at this and think "why didn't they do something like ``core.info(`Scoped Variables: ${ JSON.stringify(environmentDictionary) }`)``"?

Reply:
Yes that's exactly it.